package.json

```diff
@@ -9,19 +9,18 @@
     "preview": "vite preview",
     "fmt": "npx prettier --write *.html *.js *.svelte components/* tests/*",
     "lint": "npx eslint . --ignore-pattern=vite.config.js",
-    "test": "vite build && vitest"
+    "test": "npx playwright test"
   },
   "devDependencies": {
-    "@playwright/test": "^1.30.0",
+    "@playwright/test": "1.31.2",
```

Comment:
There was some github issue (I lost track) for a problem waiting forever for the webserver to start, seemingly fixed in this newer version